test/browser/shared/bootstrap.js

```diff
@@ -0,0 +1,154 @@
+            '--hide-scrollbars',
+            '--ignore-certificate-errors',
+            '--no-sandbox',
+            '--use-fake-ui-for-media-stream',
+            '--use-fake-device-for-media-stream',
+        ],
+        headless: HEADLESS,
+        pipe: true,
+    })
+
+    let pages = await browser.pages()
+    pages[0]._name = name
+    return {browser, pages}
+}
+
+
+/**
+ * Report a test step of `actor`.
+ * When not in HEADLESS mode, it will also pause for 2 seconds.
+ * @param {String} actor - Name of the actor.
+ * @param {String} message - Message to print.
+ */
+async function step(actor, message) {
+    console.log(`[browser] <${actor}> ${message}`)
+    if (!HEADLESS) await utils.delay(2000)
+}
+
+
+/**
+ * Take a screenshot of `browser` and write it to file.
+ * @param {Object} browser - Browser instance returned by `loginAndWizard`.
+ * @param {*} name - Name the screenshot (actor name will be prepended).
+ */
+async function screenshot({app, page}, name) {
+    if (SCREENS) {
+        await mkdirp(settings.SCREENS_DIR)
+        const filename = `${page._name}-${name}.png`
+        screenshotPath = path.join(settings.SCREENS_DIR, filename)
+        await app.screenshot({path: screenshotPath})
+    }
+}
+
+
+const login = require('./login')(settings, screenshot)
+const wizard = require('./wizard')(settings, screenshot)
+
+
+/**
+ * Start a new browser, login and complete the wizard and return the
+ * instance for further testing. Login credentials are read from the
+ * settings.
+ * @param {String} name - Name of the actor.
+ * @param {Function} onExit - Exit function registration.
+ * @param {Object} options - Options.
+ * @returns {Object} - Browser instance.
+ */
+async function loginAndWizard(name, onExit, {screens = false} = {}) {
+    await step(name, 'Opening browser')
+    let browser = await createBrowser(name)
+
+    // Keep browsers open when HEADLESS=false, this will halt the next tests
+    // and gives the developer time to debug.
+    if (HEADLESS) {
+        onExit(async () => {
+            console.log(`[browser] <${name}> Closing browser.`)
+            await browser.browser.close()
+        })
+    } else {
+        console.log('HEADLESS=false: Not closing browsers automatically.')
+    }
+
+    let page = browser.pages[0]
+    page.setViewport({height: 600, width: 500})
+
+    const uri = `http://127.0.0.1:${brand.tests.port}/index.html?test=true`
+    await page.goto(uri, {})
+
+    const me = {
+        browser: browser,
+        page: page,
+        app: await page.$('#app'),
+    }
+
+    await step(name, 'Logging in.')
+    await login(me, screens)
+
+    await step(name, 'Completing wizard.')
+    const options = await wizard(me, screens)
+    await page.click('.test-delete-notification')
+
+    // Wait until the status indicates a registered device.
+    await page.waitFor('.test-status-registered')
+
+    return Object.assign({options: options}, me)
+}
+
+
+module.exports = {
+    brand: brand,
```

Comment:
No need to type the key and the value twice when they are the same or do you prefer this format?